### ISSUE TITLE
Limit autokick/ban commands 

### ISSUE THREAD

**Issue by ChargedCreeper** [TotalFreedom/TotalFreedomMod#315]
Trolls are abusing the fact that harmful commands get players autokicked for trying to use. I suggest that such commands be changed to just return unknown command or something. As it is currently, there is a problem with trolls saying a bad command in chat, and some people immediately test the command, and get autokicked.

**Comment by JeromSar** [comment]
Yes, some of the commands should be downgraded from the auto-eject system.
